operator/cmd/mesh/testdata/profile-dump/output/all_off.yaml

```diff
@@ -289,9 +284,6 @@ spec:
           name: ingressgateway-ca-certs
           secretName: istio-ingressgateway-ca-certs
         type: LoadBalancer
-        zvpn:
```

Comment:
if this is just test data, this is probably ok. If not, @ayj should take a look. We don't want to break multicluster just as we are fixing it for 1.5.1 :). @ostromart can you confirm one way or another?

Reply:
yep, just test data

Reply:
Looks okay glancing through the changes. Only way to really is to test it.

Reply:
resolving as this is simply test data.